packages/components/infinite-scroll-area/tests/infinite-scroll-area.test.tsx

```diff
@@ -27,7 +27,9 @@ describe("<InfiniteScrollArea />", () => {
         {Array(50)
           .fill(0)
           .map((_, index) => (
-            <Fragment key={index}>{index}</Fragment>
+            <div key={index} role="article">
```

Comment:
Is there any benefit to changing this test code?

Reply:
Made corrections to comply with “Certain ARIA roles must contain particular children (aria-required-children)”